notebooks/video_clips/VideoClips.ipynb

```diff
@@ -0,0 +1,963 @@
+{
```

Comment:
Let's follow a different organization - video_object_detection/yolov4_clips/*
or something along those lines
I want to call out applications more up top. notebooks doesn't say much and you end up having helper files that are not notebooks.